cmd/argo/lint/lint_test.go

```diff
@@ -63,6 +64,16 @@ spec:
         args: ["{{inputs.parameters.message}}"]
 `)
 
+type mockWriter struct {
+	mock.Mock
+}
+
+func (mw *mockWriter) Write(buf []byte) (int, error) {
```

Comment:
minor - but you could generate your mock using `mockery` see `ContainerRuntimeExecutor` for example